### PR TITLE
Adds the packaging scripts from SS14

### PR DESCRIPTION
**Pull request by ike709**
Copies the scripts over from https://github.com/space-wizards/space-station-14/tree/master/Tools which are needed to package game builds. Mainly so players connecting through the launcher actually have a client to download.

I made minimal changes to the scripts to accommodate our renaming. I'm not 100% sure everything is functional until I've finished sorting out the test server, hence this being a draft.

### CODE REVIEW

**Comment by ike709** [comment]
It works.

**Comment by wixoaGit** [comment]
Why is a script packaging the client necessary? Doesn't the server already do that?

**Comment by ike709** [comment]
/shrug

I didn't bother to figure out the reasoning, I just know ACZ dies unless both are present.

File: Tools/package_client_build.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+}
+
+CLIENT_CONTENT_ASSEMBLIES = [
+    "OpenDreamClient",
+    "OpenDreamShared"
+]
+
+def main() -> None:
+    parser = argparse.ArgumentParser(
+        description="Packages the SS14 client.")
```
Comment (wixoaGit) [inline Tools/package_client_build.py:53]:
```suggestion
        description="Packages the OpenDream client.")
```

File: Tools/package_client_build.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+            "/v:m",
+            "/t:Rebuild",
+            "/p:FullRelease=True",
+            "/m"
+        ], check=True)
+
+    print(Fore.GREEN + "Packaging client..." + Style.RESET_ALL)
+
+    client_zip = zipfile.ZipFile(
+        p("release", "SS14.Client.zip"), "w",
```
Comment (wixoaGit) [inline Tools/package_client_build.py:104]: Would changing the name of this zip break anything?

File: Tools/package_server_build.py [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+    "pt-BR",
+    "ru",
+    "tr",
+    "zh-Hans",
+    "zh-Hant"
+]
+
+def main() -> None:
+    parser = argparse.ArgumentParser(
+        description="Packages the SS14 content repo for release on all platforms.")
```
Comment (wixoaGit) [inline Tools/package_server_build.py:98]:
```suggestion
        description="Packages the OpenDream repo for release on all platforms.")
```

File: Tools/package_server_build.py [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+            f"/p:TargetOS={platform.target_os}",
+            "/t:Rebuild",
+            "/p:FullRelease=True",
+            "/m"
+        ], check=True)
+
+        publish_client_server(platform.rid, platform.target_os)
+
+    print(Fore.GREEN + "Packaging {platform.rid} server..." + Style.RESET_ALL)
+    server_zip = zipfile.ZipFile(p("release", f"SS14.Server_{platform.rid}.zip"), "w",
```
Comment (wixoaGit) [inline Tools/package_server_build.py:174]: Would changing the name of this zip break anything?

**Review by wixoaGit — CHANGES_REQUESTED** [review]

File: Tools/package_client_build.py (hunk 2 above)
Comment (ike709) [reply Tools/package_client_build.py:104]: Yes, and I learned that the hard way. I made an issue already on the SS14 repo to debrand the tools.

File: Tools/package_server_build.py (hunk 4 above)
Comment (ike709) [reply Tools/package_server_build.py:174]: See other reply.

**Review by wixoaGit — APPROVED** [review]

**Comment by ike709** [comment]
\> Approved without merge

😩

**Pull request merged**